src/ui/public/url/redirect_when_missing.js

```diff
@@ -57,7 +57,9 @@ export function RedirectWhenMissingProvider($location, kbnUrl, Promise) {
       url += (url.indexOf('?') >= 0 ? '&' : '?') + `notFound=${error.savedObjectType}`;
 
       toastNotifications.addWarning({
-        title: 'Saved object is missing',
+        title: i18n('common.ui.url.savedObjectsIsMissingNotificationMessage', {
```

Comment:
Minor typo: `savedObjectsIsMissingNotificationMessage` (plural saved objects) should be `savedObjectIsMissingNotificationMessage` (singular saved object).